### ISSUE TITLE
Startparameter to set hibe and start string not woking (Linux)

### ISSUE THREAD

**Issue by BolverBlitz** [gekware/minecraft-server-hibernation#143]
### Current Behavior (🐛 Bug)

Running MSH with `-infohibe "§fserver status:\n §b§lHIBERNATING" -infostar "§fserver status:\n §6§lWARMING UP"` will show in minecraft: FILE CONFIG START and FILE CONFIG HIBE
Config:
```json
    "InfoHibernation": "FILE CONFIG HIBE",
    "InfoStarting": "FILE CONFIG START",
```

### Expected Behavior

It should show the string passed via the parameter

### Steps to Reproduce

Start MSH with the parameters and check in minecraft

### MSH-Version

2.4.6

### Operating system

Linux (Debian)

### Minecraft Version

1.18.2

### Minecraft Server Software

Paper

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details.

**Comment by gekigek99** [comment]
should be fixed in d3976544f2691db5c5dda17c4ee6442306f76c4c

@BolverBlitz do you confirm?

**Comment by gekigek99** [comment]
I confirm that it works, but apparently for me it was working even before... so @BolverBlitz what do you say?
